### PR TITLE
feat: add `blas/base/scabs1`

### PR DESCRIPTION
**Pull request by aman-095**
## Description

This RFC proposes to add a routine to compute the sum of the absolute value of the real and imaginary part of a single-precision complex floating-point number as defined in [BLAS](https://netlib.org/lapack/explore-html//d7/d03/group__scalar__grp.html) scalar operations. Specifically adding `@stdlib/blas/base/scabs1` is proposed.

## Related Issues

> Does this pull request have any related issues?

None.

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

### CODE REVIEW

File: lib/node_modules/@stdlib/blas/base/scabs1/benchmark/c/benchmark.length.c
Comment (kgryte) [inline lib/node_modules/@stdlib/blas/base/scabs1/benchmark/c/benchmark.length.c:1]: This file and the Fortran bench file should be renamed. The `.length` part of the basename doesn't make sense, as `scabs1` does not operate on arrays.

Comment (kgryte) [reply lib/node_modules/@stdlib/blas/base/scabs1/benchmark/c/benchmark.length.c:1]: That will also mean updating the `Makefile`.

File: lib/node_modules/@stdlib/blas/base/scabs1/benchmark/benchmark.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+// MAIN //
+
+bench( pkg, function benchmark( b ) {
+	var values;
+	var y;
+	var i;
+
+	values = [
+		new Complex64( uniform( -500.0, 500.0, options ), uniform( -500.0, 500.0, options ) ), // eslint-disable-line max-len
```
Comment (kgryte) [inline lib/node_modules/@stdlib/blas/base/scabs1/benchmark/benchmark.js]: `@stdlib/random/base/uniform` does not accept a `dtype` option. I believe this is a copy-paste error from array benchmarks. This should be addressed here and in the other JS benchmark file.

File: lib/node_modules/@stdlib/blas/base/scabs1/README.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+for ( i = 0; i < 100; i++ ) {
+    z = new Complex64( discreteUniform( -50, 50 ), discreteUniform( -50, 50 ) );
+    console.log( 'scabs1(%s) = %d', z.toString(), scabs1( z ) );
+}
+```
+
+</section>
+
+<!-- /.examples -->
+
```
Comment (kgryte) [inline lib/node_modules/@stdlib/blas/base/scabs1/README.md:163]: @aman-095 You are missing C API documentation.

**Review by kgryte — CHANGES_REQUESTED** [review]
@aman-095 This PR is close. I had to do a decent amount of clean-up, including addressing copy-paste errors. What remains is updating the benchmark files and adding the C API documentation to the README.

**Comment by kgryte** [comment]
@aman-095 Thanks for making the changes. Anything else needing to be done? If not, you can go ahead and port the changes over to `dcabs1` so we can get that one over the finish line, as well.

**Comment by aman-095** [comment]
Thanks, Athan, for reviewing. This seems fine to me. I'm also replicating this in `dcabs1` and will push the changes soon.

**Review by kgryte — APPROVED** [review]
LGTM. Had to make a few more changes. Note that the dependencies for each configuration in a manifest.json file should include only those packages which are necessary for that build. Some of them had extraneous deps or were missing others. Will merge once CI passes.

**Pull request merged**